### PR TITLE
MAINT: Adds get_ctable method to BcolzMinuteBarReader

### PR DESCRIPTION
**Pull request by yankees714**
For convenience in accessing the raw ctables.

### CODE REVIEW

File: zipline/data/minute_bars.py [hunk 1]
```diff
@@ -552,6 +552,10 @@ def _open_minute_file(self, field, sid):
 
         return carray
 
+    def get_ctable(self, sid):
+        ctable_path = os.path.join(self._rootdir, _sid_subdir_path(sid))
```
Comment (ssanderson) [inline zipline/data/minute_bars.py]: I think this is `self.sidpath`?

Comment (ssanderson) [inline zipline/data/minute_bars.py]: This looks like it's replicating the behavior of `_ensure_ctable`, except without the guaranteed initialization.  When would we want to call this without guaranteeing that the table already exists?

Comment (yankees714) [inline zipline/data/minute_bars.py]: Ahh yeah missed `self.sidpath` somehow, that should be enough for my use case.

Comment (yankees714) [inline zipline/data/minute_bars.py]: @ssanderson Realized the confusion here, the methods you mentioned above of the BcolzMinuteBarWriter, whereas I was adding `get_ctable` to BcolzMinuteBarReader, for consumers of the minute bars. But they probably should share this path creation logic.

**Comment by yankees714** [comment]
This is superseded by `BcolzMinuteBarReader.unadjusted_window`  (#1050) .

**Pull request closed without merging**